### ISSUE TITLE
Windows can't write to file

### ISSUE THREAD

**Issue by patruff** [Doriandarko/maestro#29]
Error using maestro, line 303, should have it forced to UTF-8 (crashed on Windows). Otherwise great, thanks for the framework!

**Comment by jimmc414** [comment]
change 
`with open(filename, 'w') as file:` 
to 
`with open(filename, 'w', encoding='utf-8') as file:`

**Comment by patruff** [comment]
Yup @jimmc414 I did, just didn't want to re-run it. It's just the log, for some reason it wasn't outputting the code to file (again might have been a Windows issue), but I got the code I wanted in the output text.

**Comment by emimix** [comment]
I also can't get it to create the actual files, only the folders. I'm using Windows 11

**Comment by Subin-Vidhu** [comment]
same issue, tried it on both Linux and Windows, but still no files inside the directory. Followed the following as well:

> change `with open(filename, 'w') as file:` to `with open(filename, 'w', encoding='utf-8') as file:`

still no luck...

**Comment by emimix** [comment]
I also can't get it to create the actual files, only the folders. I'm using Windows 11

**Comment by marwinsteiner** [comment]
I have the same issues, getting it to write actual files is proving to be a challenge.

**Comment by Eipckz** [comment]
Turned on notifications for this one. I also have the same issue. It seems getting the files to sucessfully write are about a 50/50. But, I think it has to do with how long your "objective" when first running it. I have instead put, "Follow the instructions given in the file provided" and I would have better sucess rates with it actually printing the results of the files. but it isn't perfect, and there is about a 60% sucess rate with this method. Of course, this does not work well if you have a file you wanted to give your chat model, but nonetheless, there is a way to do it.
